### DISCUSSION TITLE
Need to create a new query parameter in WADO RS/ WADO URI

### DISCUSSION THREAD

**Discussion by maya-a8** [dcm4che/dcm4chee-arc-light#3770] [General]
Can someone help me with telling the steps on how to create a new query parameter to the WADO RS? Or if there is any documentation that can help me get started, it would be great.

**Comment by vrindanayak** [comment]
What do you mean by 
> create a new query parameter to the WADO RS

Archive's WADO-RS services follows the [Retrieve Transaction in DICOM Part 18](https://dicom.nema.org/medical/dicom/current/output/html/part18.html#sect_10.4)
